server/tests/ingest/parser/cp_onclusive_test.py

```diff
@@ -0,0 +1,159 @@
+import json
+from cp.ingest.parser.cp_onclusive import CPOnclusiveFeedParser
+import datetime
+from dateutil.tz import tzutc
+from . import ParserTestCase
+from superdesk.metadata.item import (
+    ITEM_TYPE,
+    CONTENT_TYPE,
+    GUID_FIELD,
+    CONTENT_STATE,
+)
+import flask
+import superdesk
+from tests.ingest.parser import get_fixture_path
+from tests.mock import resources
+from unittest.mock import patch
+
+
+with open(get_fixture_path("cp_onclusive.json", "cp_onclusive")) as fp:
+    data = json.load(fp)
+
+
+class OnclusiveFeedParserTestCase(ParserTestCase):
+    parser = CPOnclusiveFeedParser()
+    provider = {}
+    app = flask.Flask(__name__)
+
+    def test_content(self):
+        with self.app.app_context():
+            with patch.dict(superdesk.resources, resources):
+                item = self.parser.parse(data)[0]
+                item["subject"].sort(key=lambda i: i["name"])
+                expected_subjects = [
+                    {
+                        "name": "Law & Order",
+                        "qcode": "88",
+                        "scheme": "onclusive_categories",
+                    },
+                    {
+                        "name": "Conflict / Terrorism / Security",
+                        "qcode": "133",
+                        "scheme": "onclusive_categories",
+                    },
+                    {
+                        "name": "Trade Conferences",
+                        "qcode": "97",
+                        "scheme": "onclusive_categories",
+                    },
+                    {
+                        "name": "Banking",
+                        "qcode": "159",
+                        "scheme": "onclusive_categories",
+                    },
+                    {
+                        "name": "Finance General",
+                        "qcode": "35",
+                        "scheme": "onclusive_categories",
+                    },
+                    {
+                        "name": "Tech - Internet, software & new media",
+                        "qcode": "50",
+                        "scheme": "onclusive_categories",
+                    },
+                    {
+                        "name": "Trade Conferences",
+                        "qcode": "148",
+                        "scheme": "onclusive_event_types",
+                    },
+                    {
+                        "name": "Cyber Security and Fraud",
+                        "qcode": "228",
+                        "scheme": "onclusive_event_types",
+                    },
+                ]
+                expected_subjects.sort(key=lambda i: i["name"])
+                self.assertEqual(item["subject"], expected_subjects)
+
+                item["anpa_category"].sort(key=lambda i: i["name"])
+                expected_categories = [
+                    {
+                        "name": "National",
+                        "qcode": "g",
+                        "is_active": True,
```

Comment:
this `is_active` we don't store in the items, only in vocabulary. but we should only use those values which are active. there is a helper in vocabularies service `.get_items`